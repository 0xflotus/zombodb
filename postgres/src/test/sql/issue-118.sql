SELECT term FROM zdb_significant_terms('so_posts', 'owner_display_name', '^.*', 'java', 5000);