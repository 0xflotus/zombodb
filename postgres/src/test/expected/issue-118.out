SELECT term FROM zdb_significant_terms('so_posts', 'owner_display_name', '^.*', 'java', 5000);
       term       
------------------
 user1249
 user104150
 user1449
 striplingwarrior
 user7197
 user39685
 martin
 cletus
 konrad rudolph
 user42242
 kevin
 user4051
 duffymo
 sergio
 user29079
 user8685
(16 rows)

